code/game/objects/effects/anomalies.dm

```diff
@@ -375,3 +394,138 @@
+
+/obj/effect/anomaly/hallucination
+	name = "hallucination anomaly"
+	icon_state = "hallucination_anomaly"
+	aSignal = /obj/item/assembly/signaler/anomaly/hallucination
+	/// Time passed since the last effect, increased by delta_time of the SSobj
+	var/ticks = 0
+	/// How many seconds between each small hallucination pulses
+	var/release_delay = 5
+
+/obj/effect/anomaly/hallucination/anomalyEffect(delta_time)
+	. = ..()
+	ticks += delta_time
+	if(ticks < release_delay)
+		return
+	ticks -= release_delay
+	var/turf/open/our_turf = get_turf(src)
+	if(istype(our_turf))
+		hallucination_pulse(our_turf, 5)
+
+/obj/effect/anomaly/hallucination/detonate()
+	var/turf/open/our_turf = get_turf(src)
+	if(istype(our_turf))
+		hallucination_pulse(our_turf, 10)
+
+/proc/hallucination_pulse(turf/location, range, strength = 50)
+	for(var/mob/living/carbon/human/near in view(location, range))
+		// If they are immune to hallucinations
+		if (HAS_TRAIT(near, TRAIT_MADNESS_IMMUNE) || (near.mind && HAS_TRAIT(near.mind, TRAIT_MADNESS_IMMUNE)))
+			continue
+
+		// Blind people don't get hallucinations
+		if (is_blind(near))
+			continue
+
+		// Everyone else
+		var/dist = sqrt(1 / max(1, get_dist(near, location)))
+		near.hallucination += strength * dist
+		near.hallucination = clamp(near.hallucination, 0, 150)
+		var/list/messages = list(
+			"You feel your conscious mind fall apart!",
+			"Reality warps around you!",
+			"Something's wispering around you!",
+			"You are going insane!",
+			"What was that?!"
+		)
+		to_chat(near, "<span class='warning'>[pick(messages)]</span>")
+
+/////////////////////
+
+/obj/effect/anomaly/delimber
+	name = "delimber anomaly"
+	icon_state = "delimber_anomaly"
+	aSignal = /obj/item/assembly/signaler/anomaly/delimber
+	immortal = TRUE
+	/// Cooldown for every anomaly pulse
+	COOLDOWN_DECLARE(pulse_cooldown)
+	/// How many seconds between each anomaly pulses
+	var/pulse_delay = 15 SECONDS
+	/// Range of the anomaly pulse
+	var/range = 5
+
+/obj/effect/anomaly/delimber/Initialize(mapload, new_lifespan)
+	. = ..()
+	if(new_lifespan)
+		immortal = FALSE //manually override the immortality lifespan
+
+/obj/effect/anomaly/delimber/anomalyEffect(delta_time)
+	. = ..()
+
+	if(!COOLDOWN_FINISHED(src, pulse_cooldown))
+		return
+
+	COOLDOWN_START(src, pulse_cooldown, pulse_delay)
+
+	delimber_pulse(src, range)
+
+/proc/delimber_pulse(atom/owner, range = 5, ignore_owner = FALSE, message_admins = FALSE)
+	var/list/mob/living/carbon/affected = list()
+	for(var/mob/living/carbon/target in range(range, owner))
+		if(!ignore_owner && target == owner)
+			continue
+		if(target.run_armor_check(attack_flag = "bio", absorb_text = "Your armor protects you from [owner]!") >= 100)
+			continue //We are protected
+
+		// Add target
+		affected += target
+
+		// Insert a random organ
+		var/obj/item/organ/picked_organ = pick(ANOMALY_DELIMBER_ZONE_ORGANS)
+		var/obj/item/organ/new_organ = new picked_organ
+		new_organ.Insert(target, TRUE, FALSE)
```

Comment:
just saying, this literally has the ability to completely remove you if it chooses any brain, it will literally qdel you when `Insert()` calls `Remove()` (which puts you in your brain and then deletes it)

Reply:
blacklisted brains, xenoegg spawners and romerol tumors